src/main/java/uc/doc/voteonsolution/VoteSDocResponseModel.java

```diff
@@ -0,0 +1,29 @@
+package uc.doc.voteonsolution;
+
+import java.time.LocalDateTime;
+
+import entities.SolutionDocument;
+
+public class VoteSDocResponseModel {
+    private final int voteTotal;
+    private final SolutionDocument solutionDoc;
+    private final LocalDateTime timestamp;
+
+    public VoteSDocResponseModel(int voteTotal, SolutionDocument solutionDoc, LocalDateTime timestamp) {
+        this.voteTotal = voteTotal;
+        this.solutionDoc = solutionDoc;
+        this.timestamp = timestamp;
+    }
+
+    public int getVoteTotal() {
+        return this.voteTotal;
+    }
+
+    public SolutionDocument getSolutionDoc() {
```

Comment:
Response models should not depend (or at least return) entities. All response models need rewritten anyway so don't worry about correcting in this PR